feat(backend): Deprecate `domain` field in favor of `domains` on SAML connections/account (#6186)
clerk/javascript: .changeset/lucky-spoons-wonder.md

---
'@clerk/backend': minor
---

> Deprecates `domain` field and introduce `domains`. Now, SAML connections support multiple domains, the `domain` field still supported but it's deprecated and will be removed on a future API version.

Reviewer: ```suggestion
Introduces support for SAML connections with multiple domains, deprecating the `domain` field from `CreateSamlConnectionParams` and `UpdateSamlConnectionParams` in favor of `domains`.
```
